### PR TITLE
Домашнее задание к спринту №19

### PR DESCRIPTION
**Pull request by Aslan4242**

### CODE REVIEW

File: app/src/main/java/com/example/playlistmaker/media/ui/MediaFragment.kt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                0 -> tab.text = getString(R.string.favorite_tracks_tab)
+                1 -> tab.text = getString(R.string.playlists_tab)
+            }
+        }
+        tabLayoutMediator.attach()
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        tabLayoutMediator.detach()
```
Comment (i-masloed) [inline app/src/main/java/com/example/playlistmaker/media/ui/MediaFragment.kt:40]: Рекомендация:  tabLayoutMediator.detach() стоит вызвать до super.onDestroy()

File: app/src/main/java/com/example/playlistmaker/media/ui/MediaFragment.kt [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.example.playlistmaker.R
+import com.example.playlistmaker.databinding.FragmentMediaBinding
+import com.google.android.material.tabs.TabLayoutMediator
+
+class MediaFragment : Fragment() {
+    private lateinit var binding: FragmentMediaBinding
```
Comment (i-masloed) [inline app/src/main/java/com/example/playlistmaker/media/ui/MediaFragment.kt:13]: Рекомендация: Здесь стоит  организовать биндинг также, как и в FavouritesFragment

File: app/src/main/java/com/example/playlistmaker/media/ui/MediaFragment.kt (hunk 1 above)
Comment (Aslan4242) [reply app/src/main/java/com/example/playlistmaker/media/ui/MediaFragment.kt:41]: поправил

File: app/src/main/java/com/example/playlistmaker/media/ui/MediaFragment.kt (hunk 2 above)
Comment (Aslan4242) [reply app/src/main/java/com/example/playlistmaker/media/ui/MediaFragment.kt]: поправил

**Comment by Aslan4242** [comment]
@i-masloed в заданиях нет такого требования, сделать работающий свитч переключения темной темы
